src/assets/scss/gds/elements/partials/_panels.scss

```diff
@@ -6,6 +6,7 @@
   @extend %contain-floats;
   clear: both;
   border-left-style: solid;
+  border-left-width: 5px;
```

Comment:
Are you sure this won't effect anything in the app?

Reply:
The `.panel` does not define its own border width.  I'd say this is required and has no ill effect.

A width should probably be defined on the `.panel` as it gets computed as **4.5px** otherwise (getting its value all the way from from reset!) . 

The actual border width isn't defined anywhere until you add classes: `panel-border-wide` or `panel-border-narrow`. 

![screen shot 2018-07-30 at 13 44 40](https://user-images.githubusercontent.com/7467069/43398035-bf2af22e-93fe-11e8-917d-9d96d7e113ae.png)